### ISSUE TITLE
[BUG][NOWEB] -  Docker container is being crashed again and again on Render

### ISSUE THREAD

**Issue by ravikant-pal** [allburov/whatsapp-http-api#645]
### Describe the bug

Docker container is being crashed again and again on Render am I doing wrong or there is bug

### Version

```json
{
  "version": "2024.11.5",
  "engine": "NOWEB",
  "tier": "CORE",
  "browser": "/usr/bin/chromium"
}
```

## Steps

**To Reproduce** Steps to reproduce the behavior:

1. Go to render.com
2. Signup and Click on New → Web Service → Existing Image
3. put image name 'devlokepro/waha'
4. add environment variable `WHATSAPP_DEFAULT_ENGINE` =NOWEB
5. deploy and then session is coming as stoped try to start 
6. get the error in logs of render.

### Expected behavior

should work fine.

### Docker Logs

```
/app/node_modules/rxjs/dist/cjs/internal/util/reportUnhandledError.js:13
            throw err;
            ^
TypeError: Cannot read properties of undefined (reading 'get')
    at SessionManagerCore.getSessionEvent (/app/dist/core/manager.core.js:152:29)
    at /app/dist/core/abc/manager.abc.js:39:64
    at Array.map (<anonymous>)
    at SessionManagerCore.getSessionEvents (/app/dist/core/abc/manager.abc.js:39:44)
    at WebsocketGatewayCore.handleConnection (/app/dist/core/api/websocket.gateway.core.js:40:14)
    at Object.next (/app/node_modules/@nestjs/websockets/web-sockets-controller.js:77:47)
    at ConsumerObserver.next (/app/node_modules/rxjs/dist/cjs/internal/Subscriber.js:113:33)
    at Subscriber._next (/app/node_modules/rxjs/dist/cjs/internal/Subscriber.js:80:26)
    at Subscriber.next (/app/node_modules/rxjs/dist/cjs/internal/Subscriber.js:51:18)
    at /app/node_modules/rxjs/dist/cjs/internal/operators/distinctUntilChanged.js:18:28
    at OperatorSubscriber._this._next (/app/node_modules/rxjs/dist/cjs/internal/operators/OperatorSubscriber.js:33:21)
    at Subscriber.next (/app/node_modules/rxjs/dist/cjs/internal/Subscriber.js:51:18)
    at /app/node_modules/rxjs/dist/cjs/internal/Subject.js:69:34
    at Object.errorContext (/app/node_modules/rxjs/dist/cjs/internal/util/errorContext.js:22:9)
    at Subject.next (/app/node_modules/rxjs/dist/cjs/internal/Subject.js:59:24)
    at WebSocketServer.<anonymous> (/app/node_modules/@nestjs/websockets/web-sockets-controller.js:61:24)
```

### Screenshots

<img width="1651" alt="image" src="https://github.com/user-attachments/assets/a96ecfa1-a2f6-47cd-81fb-e6abbd45532d">

<img width="1647" alt="image" src="https://github.com/user-attachments/assets/0fe20813-50ee-47d6-a067-b72385b21cf6">

**Comment by mrbarletta** [comment]
Having the same issue with docker using `WEBJS (2024.11.5)`

**Comment by abdoel1993** [comment]
Having the same issue with docker using WEBJS (2024.11.5)

**Comment by devlikepro** [comment]
Hi! Thank you a lot for the details!
Fixed in **2024.11.6** :rocket: 

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
